remoteit/desktop
# [DESK-860] Check host and port

### Description

Provide `responding` status for target services setup

### Test plan


1. Sign in
2. Go to device
3. Click en Edit device
4. Click en one service
5. Edit hostname and/or port
6. you should see an icon with the result of the check

### Ticket

https://remoteit.atlassian.net/browse/DESK-860

<https://remoteit.atlassian.net/browse/TICKET_NUMBER_HERE>
